Vagrantfile

```diff
@@ -64,8 +73,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       ansible.host_vars = {
         "all" => { "ansible_ssh_user" => $vagrantUser }
       }
-      ansible.extra_vars = { "islandora_distro" => $vagrantBox,
-                             "islandora_profile" => $drupalProfile }
+      ansible.extra_vars = { "islandora_distro" => "ubuntu/focal64",
+                             "islandora_profile" => $drupalProfile,
+                             "islandora_build_base_box" => $buildBaseBox }
+      ansible.raw_ssh_args = [ "-o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o ControlMaster=auto -o ControlPersist=60s" ]
```

Comment:
So... before this, i could deploy the playbook on a remote server with `ansible-playbook -i inventory/baseline playbook.yml -e "islandora_distro=ubuntu/focal64"` (where `baseline` is my customized inventory). 

Trying to do that with this modified playbook (also passing 'islandora_base_box') is causing an access denied error:

```
fatal: [default]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: no such identity: /home/rlefaive/.ssh/id_rsa: No such file or directory\r\nroot@147.182.144.21: Permission denied (publickey).", "unreachable": true}
```

I am guessing it has to do with this line?